### ISSUE TITLE
add fullscreen option: remove 758px width limitation on display mode

### ISSUE THREAD

**Issue by Renato-Rodrigues** [hedgedoc/hedgedoc#2828]
### Which part of the project should be enhanced?

markdown visualization width screen state use

### Describe the solution you'd like

I would like to have an option, maybe a toggle, to use the entire screen state on the markdown document visualization instead of limiting the view to 758px wide.

If you use tables with multiple columns for example, the current width limitation only displays about four columns and requires back and forth using the horizontal scroll to see the remaining content.

### Describe alternatives you've considered

I am using a hacky solution and disabling the "max-width: 758px;" property on the id="doc" element in the browser to get a full view of my documents and tables. 
This is an approximation of the desired result if the option to toggle fullscreen view would be added in the future.

**Comment by ErikMichelson** [comment]
This sounds similar to #25

**Comment by Avi98** [comment]
We can toggle the editor and renderer in V2

https://user-images.githubusercontent.com/26133749/213416308-c0a84fb4-6421-4144-8926-3e07de34dcd4.mov

**Comment by mrdrogdrog** [comment]
This has already been able in HedgeDoc 1. This issue is a about the width of the content in the renderer on the right side .

**Comment by SeanMooney** [comment]
this limitation is still there in 2024
its frustrating as i was trying to use hedgedoc instead of eatherpad for taking notes when manually reviewing and testing a proposed patch to the opensource project i work on.

i was using code blocks to embed the output of commands and logs into the document but they were needless confined to 758px resulting in horizontal scroll bars

hedgedoc is nice but with this limitation i feel like etherpad is better for notes like this and a static site generator like hugo is going to be better for blogs

that severely reducice the utility of hedgdoc in may respects as a result.

**Comment by jakubgs** [comment]
I agree, I think the width of the document content should be configurable, either in the UI or in the service config file. Many tables become unreadable in HackMD and HedgeDoc because they get squashed down `758px`.
